presto-main/src/main/java/com/facebook/presto/server/security/SpnegoFilter.java

```diff
@@ -165,7 +165,8 @@ public Principal getUserPrincipal()
                         return;
                     }
                 }
-                catch (GSSException e) {
+                catch (Exception e) {
+                    LOG.debug(e, "Authentication failed for token %s", parts[1]);
```

Comment:
@electrum, thanks for the comments. I understand that in general Log-and-throw is not recommended, here I just want to log the token content as it will not pop up along with the exception, how about:
LOG.debug("Authentication failed for token %s", parts[1])?